tests/no-json-module-file-ext/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "ES5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmitOnError": false,
+    "jsx": "react",
+    "allowJs": true,
+    "baseUrl": "src"
+  }
+}
```

Comment:
Need a blank line at the end here

Reply:
Sure. fwiw, this was copy pasted from `imports-test`.

Reply:
Fixed